example/multistep/keras_train.py

```diff
@@ -0,0 +1,103 @@
+import click
```

Comment:
Nit: maybe we should call this train_keras instead of keras_train to make it a valid phrase, but feel free to ignore this

Reply:
Ah, now I remember why I can't call it keras -- it then shows the keras package in Python 2.